4-Classification/4-Applied/translations/assignment.az.md

```diff
@@ -0,0 +1,11 @@
+# Tövsiyyə sistemi yaradın
+
+## Təlimatlar
+
+Bu dərslərdəki təcrübələrə əsasən, Onnx Runtime və ya çevrilmiş bir Onnx modeli istifadə edərək JavaScript əsaslı veb tətbiqi necə yaratmağı artıq bilirik. Bu dərsdəki məlumatlardan və ya başqa mənbədən (zəhmət olmasa, mənbəni göstərməklə) məlumatları istifadə edərək yeni bir tövsiyyəçi yaratmaqla təcrübələr aparın. Fərqli şəxsiyyət xüsusiyyətlərinə görə müəyyən bir ev heyvanı tövsiyyəçisi və ya şəxsin əhvalına uyğun musiqi janrı tövsiyyə edən bir sistem yarada bilərsiniz. Kreativ olun!
+
+## Rubrika
+
+| Kriteriya | Nümunəvi                                                              | Adekvat                              | İnkişafa ehtiyyacı olan                 |
+| -------- | ---------------------------------------------------------------------- | ------------------------------------- | --------------------------------- |
+|          | Veb tətbiq və ya notbuk verilib Hər ikisi istifadəyə hazırdır. | İkisindən biri əksikdir və ya qüsurludur. | Hər ikisi ya əksikdir, ya da qüsurludur. |
```

Comment:
```suggestion
|          | Veb tətbiq və ya notbuk verilib. Hər ikisi istifadəyə hazırdır. | İkisindən biri əskikdir və ya qüsurludur. | Hər ikisi ya əskikdir, ya da qüsurludur. |
```

Reply:
@muradtries thanks for detailed review and corrections. all done 👍